script/build

```diff
@@ -1,24 +1,20 @@
 #!/bin/bash
 set -e
 
-if [ -z "$1" ]; then
-    OS_PLATFORM_ARG=(-os="darwin linux windows")
-else
-    OS_PLATFORM_ARG=($1)
-fi
-
-if [ -z "$2" ]; then
-    OS_ARCH_ARG=(-arch="386 amd64 arm")
-else
-    OS_ARCH_ARG=($2)
-fi
+BUILD_IMAGE_NAME="docker-machine-build"
+OS_PLATFORM_ARG=${GOOS:-"darwin linux windows"}
```

Comment:
Want to actually update this variable to `GOOS` instead of `OS_PLATFORM_ARG` for consistency / ease of mental burden's sake, then I'll be ready to merge this.